app/views/devise/sessions/new.html.erb

```diff
@@ -1,10 +1,11 @@
+<% content_for :title do %>
```

Comment:
is it because the yield is showing the title, so the <h2>Sign in</h2> is omitted?

Reply:
you got it! @Rainesisu